packages/shared-components/components/OrganizationSelector/OrgSelector.tsx

```diff
@@ -0,0 +1,59 @@
+import "./orgSelector.css";
+
+export type OrganizationSelectorItem = {
+  name: string;
+  id: string;
+};
+
+export type OrganizationSelectorProps = {
+  title: string;
+  actionTitle: string;
+  organizations: OrganizationSelectorItem[];
```

Comment:
```suggestion
  organizations: [OrganizationSelectorItem, ...OrganizationSelectorItem[]];
```
In this way the array must be at least long 1, given that you are using the `0` index in the code